src/client/storage/uploader.ts

```diff
@@ -90,3 +100,21 @@ export const uploadLoop = async () => {
 
     setTimeout(uploadLoop, UPLOAD_LOOP_INTERVAL);
 };
+
+export const chunkValidatorLoop = async () => {
+
+    const allCompletedChunks: any[] = await Chunk.allBy('ul_status', CHUNK_UPLOAD_STATUS.VALIDATING, false);
+    allCompletedChunks.forEach(async (chunk) => {
```

Comment:
I would propose to limit the array length here. If for some reason there is a hundred chunks to verify, we are going to request them all at the same time which may be not efficient.